src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -428,6 +429,10 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 		return activeCodeEditor?.getModel()?.getLanguageIdentifier().language;
 	}
 
+	get activeEditorContextKeyService(): IContextKeyService | undefined {
+		return this.activeEditorPane?.scopedContextKeyService;
```

Comment:
Shouldn't this fallback to use the scoped context key service of the active group? If you check how `invokeWithinEditorContext` used to be:

```
		const activeTextEditorControl = this.activeTextEditorControl;	
		if (isCodeEditor(activeTextEditorControl)) {	
			return activeTextEditorControl.invokeWithinContext(fn);	
		}	

		const activeGroup = this.editorGroupService.activeGroup;	
		if (activeGroup) {	
			return activeGroup.invokeWithinContext(fn);	
		}	

		return this.instantiationService.invokeFunction(fn);
```

If we end up not needing this anymore then I would actually think we can drop `activeEditorContextKeyService` altogether and let people use `activeEditorPane.scopedContextKeyService` directly.